locales/pt.json

```diff
@@ -89,127 +89,127 @@
  	"help": "ajuda",
  	"quit": "sair",
- 	"Type": "Type",
- 	"to see the list of available commands": "to see the list of available commands",
- 	"Asset Pipeline": "Asset Pipeline",
- 	"Ethereum node detected": "Ethereum node detected",
- 	"Deployment Done": "Deployment Done",
- 	"Looking for documentation? You can find it at": "A procura de Documentacao? pode encontra-la em",
- 	"Ready": "Ready",
- 	"tip: you can resize the terminal or disable the dashboard with": "tip: you can resize the terminal or disable the dashboard with",
- 	"finished building": "finished building",
- 	"Done": "Done",
- 	"Cannot upload: {{platform}} node is not running on {{url}}.": "Cannot upload: {{platform}} node is not running on {{url}}.",
- 	"try \"{{ipfs}}\" or \"{{swarm}}\"": "try \"{{ipfs}}\" or \"{{swarm}}\"",
- 	"finished deploying": "finished deploying",
- 	"finished building DApp and deploying to": "finished building DApp and deploying to",
- 	"IPFS node detected": "IPFS node detected",
- 	"IPFS node is offline": "IPFS node is offline",
- 	"not found or not in the path. Guessing %s for path": "not found or not in the path. Guessing %s for path",
- 	"adding %s to ipfs": "adding %s to ipfs",
- 	"DApp available at": "DApp available at",
- 	"error uploading to ipfs": "error uploading to ipfs",
- 	"successfully uploaded to ipfs": "successfully uploaded to ipfs",
- 	"Error while loading the content of ": "Error while loading the content of ",
- 	"error compiling for unknown reasons": "error compiling for unknown reasons",
- 	"error compiling. There are sources available but no code could be compiled, likely due to fatal errors in the solidity code": "error compiling. There are sources available but no code could be compiled, likely due to fatal errors in the solidity code",
- 	"Swarm node detected...": "Swarm node detected...",
- 	"Swarm node is offline...": "Swarm node is offline...",
- 	"deploying to swarm!": "deploying to swarm!",
- 	"adding %s to swarm": "adding %s to swarm",
- 	"error uploading to swarm": "error uploading to swarm",
- 	"successfully uploaded to swarm": "successfully uploaded to swarm",
- 	"Vyper exited with error code ": "Vyper exited with error code ",
- 	"Execution returned no result": "Execution returned no result",
- 	"compiling Vyper contracts": "compiling Vyper contracts",
- 	"Webserver is offline": "Webserver is offline",
- 	"stopping webserver": "stopping webserver",
- 	"a webserver is already running at": "a webserver is already running at",
- 	"no webserver is currently running": "no webserver is currently running",
- 	"couldn't find file": "couldn't find file",
- 	"errors found while generating": "errors found while generating",
+ 	"Type": "Tipo",
+ 	"to see the list of available commands": "para ver a lista de comandos disponíveis",
+ 	"Asset Pipeline": "Fluxo de recursos",
+ 	"Ethereum node detected": "Nó Ethereum detectado",
+ 	"Deployment Done": "Publicação feita",
```

Comment:
I think `completa` is probably better than `feita` here but I'm not 100% sure. @iurimatias?